ext/liquid_c/parser.c

```diff
@@ -184,12 +184,19 @@ void init_liquid_parser(void)
     idEvaluate = rb_intern("evaluate");
 
     cLiquidRangeLookup = rb_const_get(mLiquid, rb_intern("RangeLookup"));
+    rb_global_variable(&cLiquidRangeLookup);
+
     cRange = rb_const_get(rb_cObject, rb_intern("Range"));
+    rb_global_variable(&cRange);
+
     cLiquidVariableLookup = rb_const_get(mLiquid, rb_intern("VariableLookup"));
+    rb_global_variable(&cLiquidVariableLookup);
 
     VALUE cLiquidExpression = rb_const_get(mLiquid, rb_intern("Expression"));
+    rb_global_variable(&cLiquidExpression);
```

Comment:
Looks like this is another local variable that we accidentally called `rb_global_variable` on.

Reply:
Yes, I'd noticed these too, but wanted to see if I was misreading the situation on the single one above. Thanks for handling all of these.